docs/testplan/BGP-Suppress-FIB-Pending-test-plan.md

```diff
@@ -0,0 +1,173 @@
+# BGP Suppress FIB Pending Test Plan
+
+## Related documents
+
+| **Document Name** | **Link** |
+|-------------------|----------|
+| BGP Suppress FIB Pending HLD | [https://github.com/stepanblyschak/SONiC/blob/bgp-suppress-fib-pending/doc/BGP/BGP-supress-fib-pending.md]|
+
+## Overview
+
+As of today, SONiC BGP advertises learnt prefixes regardless of whether these prefixes were successfully programmed into ASIC.
+While route programming failure is followed by orchagent crash and all services restart, even for successfully created routes there is a short period of time when the peer will be black holing traffic.
+Also, in the following scenario, a credit loop occurs:
+###### Figure 1. Use case scenario
+
+![Use case scenario](Img/Bgp_Suppress_FIB_Pending_Use_Case.png)
```

Comment:
could we add stress and performance test?
1. Stress, to announce and withdraw 1000 routes for 5 or 10 times, then check the routes consistency. 
2. performance, what's the delay we should  expect here for like 1000 routes?

Reply:
The QA team is responsible for stress and performance test.

Reply:
Hi @StormLiangMS, I checked the test_bgp_update_timer.py script, it's the existing test for validating dut performance on handling single bgp update(single route for one time and loop test for 5 routes). 
I have two concerns on using exabgp and tcpdump to implement performance test:
1. How to send a large scale bgp updates in a short time by exabgp?
2. If it could, then, using tcpdump to capture ingress and egress 1K bgp updates at the same time, does the tcpdump capture result reliable enough?
Please share your insights.

Reply:
@echuawu  I think it is ok to have separate PR for performance and stress ones, could you open an issue to track this ask?

Reply:
@StormLiangMS , thank you, and the issue has been created to track the new PR: https://github.com/sonic-net/sonic-buildimage/issues/15194

Reply:
@echuawu could you add this new PR info to the https://github.com/sonic-net/SONiC/pull/1103 to make sure it get well tracked.

Reply:
Hi @StormLiangMS , Sure done.